### ISSUE TITLE
Python 3.10 compatiblity

### ISSUE THREAD

**Issue by kevinvalk** [syrusakbary/aiodataloader#22]
```
  File "/app/src/gateway/graphql/project.py", line 118, in <module>
    from aiodataloader import DataLoader
  File "/venv/lib/python3.10/site-packages/aiodataloader.py", line 2, in <module>
    from collections import Iterable, namedtuple
ImportError: cannot import name 'Iterable' from 'collections' (/usr/local/lib/python3.10/collections/__init__.py)
```

See (for example) https://github.com/Kozea/pygal/commit/7796f14c80fe82b9435cdc33b7d2939c8331d649 for potential fix

**Comment by markedwards** [comment]
This is duplicate of issue #16.  All that needs to happen is to push a release.

**Comment by markedwards** [comment]
Resolved as of release 0.2.1.
